jsonschema-core.md

```diff
@@ -986,13 +977,7 @@ refuse to process that schema.
 
 Implementations that do not support a vocabulary that is optionally used by a
 schema SHOULD proceed with processing the schema. The keywords will be
-considered to be unrecognized keywords as addressed by {{unrecognized}}. Note
-that since the recommended behavior for such keywords is to collect them as
-annotations, vocabularies consisting only of annotations will have the same
-behavior when used optionally whether the implementation supports them or not.
-This allows annotation-only vocabularies to be supported without custom code,
-even in implementations that do not support providing custom code for extension
-vocabularies.
+considered to be unrecognized keywords as addressed by {{unrecognized}}.
```

Comment:
This section is now contradictory.

> SHOULD proceed with processing the schema.

But considered unrecognized keywords...

> mplementations MUST refuse to process schemas which contain keywords they do not recognize, or that they recognize but do not support.

Are we really wanting to say that optinal support of a vocabulary is not something we want to support anymore?

Reply:
Yes. This is a contradiction, which I've brought up in a discussion elsewhere (will provide the link shortly).

Reply:
I would be happier if there was an Issue which identifies the inconsistency which is part of the stable release milestone. I'm fine if #1510 resolves that Issue.
I'm just hesitent to approve a PR with a known contradiction **unless** there's a clear Issue to resolve it that likely won't be missed.

Reply:
There _is_ the discussion I linked.  It describes the problem in extreme detail, and I keep linking it in all the places that involves vocabs.

Reply:
The discussion isn't going to get lost, and it doesn't need to be addressed before the spec releases, or even before #1510 merges.  It doesn't need to be tracked on a project board or be associated with a milestone.

Reply:
OK. I guess given #1510 is part of the milestone, I'm less concerned.